scripts/arangodb-docker-container-setup/docker-container-scripts/test-data/testData.json

```diff
@@ -525,7 +525,7 @@
                     "allowedUserIds": [],
                     "allowedGroupIds": []
                 },
-                "audioFilename": "4859.mp3",
```

Comment:
Not gonna work. The filename is appended to the `baseDigitalAssetURL` to build a full, (supposedly) valid audio URL.

Reply:
Not sure I get what you're saying here?

Reply:
We'll leave this issue of having the full url as the audioFilename for now.  This is fixed for the time being.